neon_data_models/models/api/llm.py

```diff
@@ -165,8 +165,10 @@ def to_completion_kwargs(self, mq2role: dict = None) -> dict:
         history = self.messages[-2*self.max_history:]
         for msg in history:
             msg["role"] = mq2role.get(msg["role"]) or msg["role"]
-        history.insert(0, {"role": "system",
-                           "content": self.persona.system_prompt})
+        if self.persona.system_prompt:
+            history.insert(0, {"role": "system",
+                               "content": self.persona.system_prompt})
+        history.append({"role": "user", "content": self.query})
```

Comment:
This is a proper fix to get right reply responce from LLM
But then [this line](https://github.com/NeonGeckoCom/brainforge-llm-service/pull/1/files#diff-b70bd2458d96cb8e20a8ce851adc1d6bef170bb10d65d2d32cfc28a1319def5bR72) need to be fixed

As it will have doubled `user` message